Company tags (#1315)
codatio/codat-docs: docs/using-the-api/managing-companies.md

```
</TabItem>

<TabItem value="java" label="Java">

```go
CompanyRequestBody req = CompanyRequestBody.builder()
    .name("Toft stores")
    .description("Need a loan for refurb.")
    .build();

CreateCompanyResponse companyCreatedRes = platformClient.companies().create()
    .request(req)
    .call();

if (companyCreatedRes.company().isPresent()) {
    // handle response
}
```
</TabItem>

</Tabs>

:::caution Retain the company ID

The `id` property that you receive in the response is the unique Codat identifier for this company. **We recommend that you retain it for future reference.**
:::

:::note Company name

The name of the company helps you identify the company in the Codat Portal and doesn't have to be unique. Make sure to [avoid forbidden characters](/core-concepts/companies).
:::

> #### Add metadata to a company 

Reviewer: oooh I hate this page 😿 your update is fine, just the whole page needs love and attention

Author: Do you want me to create a user story to fix this?

Reviewer: nope, just complaining :)

> You can store additional information about the company using the `tags` object. These tags could include anything from setting a foreign key, defining an operating region, to specifying details about the financial services a company has requested. You can add up to 10 tags when using the [Create company](/platform-api#/operations/create-company) endpoint or the [Update company](/platform-api#/operations/update-company) endpoint.

Reviewer 2: ```suggestion
You can store additional information about a company using the `tags` object. These tags could include anything from setting a foreign key, defining an operating region, to specifying details about the financial services a company has requested (see the examples below). 

You can add up to 10 tags per company. You can add tags with the [Create company](/platform-api#/operations/create-company) and [Update company](/platform-api#/operations/update-company) endpoints.
```
